Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift

```diff
@@ -1278,11 +1279,7 @@ public struct RenderNodeTranslator: SemanticVisitor {
                 )
             }
             
-            if sections.isEmpty {
-                return nil
-            } else {
-                return sections
-            }
+            return sections
```

Comment:
The change here is to explicitly emit an empty array instead of a nil array so that we don't fallback to the Swift variant if the sections for the Objective-C variant are empty?

Can we also remove the `?? .init(defaultValue: [])` then or is it still needed?